packages/base/src/utils/network.ts

```diff
@@ -107,8 +111,10 @@ const chainIds: { [key in Network]: number } = {
   fuji: 0xa869,
   optimism: 10,
   'optimism-kovan': 69,
+  'optimism-goerli': 421613,
   arbitrum: 42161,
   'arbitrum-rinkeby': 421611,
+  'arbitrum-goerli': 421611,
```

Comment:
this is something other than what `arbitrum-rinkeby` uses I'm guessing

Reply:
Thanks Dan! I can't believe how distracted I am rn